docs/application/dotnet/guides/user-interface/nui/overview.md

```diff
@@ -63,7 +63,7 @@ To be able to use NUI in your applications, you must become familiar with the fo
 
 -   **Layer**: Layers allow you to overlay groups of views on top of each other.
 
-To get started with NUI development, see [NUI Quick Start](../../get-started/nui/first-app.md) page.
+To get started with NUI development, see [NUI Quick Start](../../../get-started/first-app.md) page.
```

Comment:
Just a question. As the link is changed from NUI get started to just .NET get started,
don't we need to trim this sentence? This sentence is focusing on NUI Quick start.

Reply:
first-app in dotnet/get-started describes nui app, because we want to show that NUI is the main .Net framework for UI development.